OWNERS

```diff
@@ -1,12 +1,14 @@
 # See the OWNERS docs at https://go.k8s.io/owners
 
 reviewers:
+- aojea
```

Comment:
Can you move `MushuEE` and `spiffxp` to emeritus from reviewers and approvers? Also, can you add me as an approver? I can merge this PR manually once its ready

Reply:
done